packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -125,9 +121,14 @@ export const RelationInputDataManager = ({
   };
 
   const handleOpenSearch = () => {
+    let idsToInclude;
+
+    if (!isCreatingEntry) {
+      idsToInclude = relationsFromModifiedData?.disconnect?.map((relation) => relation.id);
+    }
+
     searchFor('', {
-      idsToInclude:
-        !isCreatingEntry && relationsFromModifiedData?.disconnect?.map((relation) => relation.id),
+      idsToInclude,
```

Comment:
@ronronscelestes if we send `false` the backend responds with an error. I think we should either omit the property from the query (`undefined`) or send an array.

Reply:
you're right I've fixed it [here](https://github.com/strapi/strapi/pull/14527/commits/a457efe802a1b1692b9f2d8f0a5edb86bb3a6cfb) but I should have fixed it directly in `feature/relations-main-view`, on it!